.release-please-manifest.json

```diff
@@ -1,20 +1,20 @@
 {
-  ".": "2.2.0",
+  ".": "2.3.0",
   "packages/api-explorer": "0.9.29",
   "packages/code-editor": "0.1.19",
   "packages/extension-api-explorer": "22.2.1",
   "packages/extension-sdk": "22.2.0",
   "packages/extension-sdk-react": "22.2.1",
```

Comment:
I think this version of release-please doesn't quite work for us yet. This package (and many others) should have bumped to 22.4.0 but they didn't

Reply:
No they shouldn't. The config explicitly defines their behavior to be conventionally versioned. Release-please v3 does not change versioning behavior. https://github.com/looker-open-source/sdk-codegen/blob/main/release-please-config.json#L8